frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -74,7 +81,79 @@ function mapStateToProps(state, { fields = [] }) {
   };
 }
 
-function FieldValuesWidgetInner({
+interface ValuesResponse {
```

Comment:
it looks like `ParameterValues` from `frontend/src/metabase-types/api/parameters.ts`, can we use it here?